.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: scala
+script:
+  - sbt test
```

Comment:
Please, make it `+test` so it goes through all supported Scala versions.

Reply:
sure - also fixed issues when running with +test